.teamcity/_self/projects/MarTech.kt

```diff
@@ -0,0 +1,131 @@
+package _self.projects
+
+import Settings
+import _self.bashNodeScript
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.perfmon
+import jetbrains.buildServer.configs.kotlin.v2019_2.failureConditions.BuildFailureOnMetric
+import jetbrains.buildServer.configs.kotlin.v2019_2.failureConditions.failOnMetricChange
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.schedule
+
+object MarTech : Project({
+	id("MarTech")
+	name = "MarTech"
+	description = "Tasks run by MarTech."
+
+	params {
+		param("docker_image", "%docker_image_e2e%")
+	}
+
+	buildType(ToSAcceptanceTracking)
+})
+
+object ToSAcceptanceTracking: BuildType ({
+	name = "ToS Acceptance Tracking"
+	description = "Captures screenshots of locations where Terms of Service are shown."
+
+
+	artifactRules = """
+		screenshots => screenshots
+	""".trimIndent()
+
+	vcs {
+		root(Settings.WpCalypso)
+		cleanCheckout = true
+	}
+
+	params {
+		param("env.NODE_CONFIG_ENV", "test")
+		param("env.PLAYWRIGHT_BROWSERS_PATH", "0")
+		param("env.TEAMCITY_VERSION", "2021")
+		param("env.HEADLESS", "false")
+		param("env.LOCALE", "en")
+	}
+
+	steps {
+		bashNodeScript {
+			name = "Prepare environment"
+			scriptContent = """
+				# Install deps
+				yarn workspaces focus wp-e2e-tests @automattic/calypso-e2e
+
+				# Build packages
+				yarn workspace @automattic/calypso-e2e build
+			""".trimIndent()
+			dockerImage = "%docker_image_e2e%"
+		}
+
+		bashNodeScript {
+			name = "Capture screenshots"
+			scriptContent = """
+				# Configure bash shell.
+				shopt -s globstar
+				set -x
+
+				# Enter testing directory.
+				cd test/e2e
+				mkdir temp
+
+				# Decrypt config
+				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%E2E_CONFIG_ENCRYPTION_KEY%"
+
+				# As noted above, Calypso E2E build configuration exports the URL
+				# environment variable in the Run tests step. Therefore, the export
+				# for NODE_CONFIG variable has to be done here instead of
+				# within the Kotlin DSL as a param() value.
+				export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL/}\"}"
```

Comment:
Where is this coming from in this case? I believe this, in the Calypso E2E builds, is coming from that bash script that fetches the Calypso live URL. It doesn't look like we are using that here, so we probably need to expose `URL` as a `param()` right?

Reply:
Ah yes, good catch. I think otherwise the value in `default.json` gets used, which is production (`https://wordpress.com`).

In this case, I'm assuming MarTech wants this to be run on production. @niranjan-uma-shankar, is my assumption true?

If the intended target is production, then we won't need the this line at all.

Reply:
> In this case, I'm assuming MarTech wants this to be run on production. @niranjan-uma-shankar, is my assumption true?

Correct, we want to run this on production.